lighthouse-core/report/html/templates.html

```diff
@@ -857,6 +856,26 @@
   </div>
 </template>
 
+<template id="tmpl-lh-3p-filter">
+  <style>
+    .lh-3p-filter {
+      background: var(--lh-table-higlight-bg);
+      color: var(--medium-75-gray);
```

Comment:
`--item-bg` instead to match https://yuinchien.github.io/sandbox/lighthouse/

name isn't great but will fix that later.